_posts/2023-08-20-my-lfx-mentorship-experience-with-the-cncf-strimzi-community.md

```diff
@@ -0,0 +1,170 @@
+
+This project got my attention among all the projects because I had practical experience with MQTT from my undergraduate program.
+I learned the basic concepts of Apache Kafka from the Cloud Computing and Distributed System course at IIIT Delhi.
+
+### The cover letter and the résumé
+
+After understanding what I had to do and what the project was about, I had to prepare my cover letter and resume.
+You need to submit these essential documents while applying to the LFX mentorship.
+I ensured my cover letter and resume were well-aligned with the project by providing details about my skills and experience with the recommended skills.
+
+But just after submitting my application, I noticed that one of the mentors for my project commented on the weak selection process.
+He was asking if they could ask for a proposal just like in Google Summer of Code.
+
+<!--Image here -->
+
+This scared me :) But I was confident I could provide a good proposal because of my previous experience with the underlying technologies.
+
+On the same day, I got my first interaction with my mentors, I would never think it wouldn’t be the last one.
+
+<!--Image here -->
+
+They gave me two days to elaborate on the proposal, and to be honest, it was the first time I saw myself as a real engineer.
+
+The first task was understanding the real problem and why we would need to build such a bridge, so I researched that.
+To design the solution, my first intuition was to relate my college project to the project.
+Just to get a rough idea of how it is very similar to what I was building in college:
+
+<!--Image here -->
+
+After that, I started to think about the architecture of the solution, and I came up with this:
+
+<!--Image here -->
+
+To be even closer to a logical solution, the win-win was to study and understand how Strimzi HTTP-Kafka Bridge works and how it is deployed on Kubernetes. 
+After reading the Strimzi HTTP-Kafka Bridge documentation, I was able to design a Bridge tackling all the issues. 
+Of course, it was not perfect, and this is where discussions and critical thinking helped me.
+
+### The mentors feedback
+
+After submitting my proposal, I got feedback from my mentors, and they were very positive about it.
+They had a lot of comments asking about my approach, and discussing and resolving the conversions was, of course, part of my win-win.
+At some point, I thought I wouldn’t be selected because my initial approach was mentioning unnecessary things like a full MQTT broker.
+
+The feedback from my mentors allowed me to rethink and defend my approach, proving that I had a basic understanding and was keen to work on the project.
+
+## The great day
+
+After so many discussions on the right approach, feedback, etc., I was not sleeping waiting for my results. 
+My first result was for the CNCF- Landing Page Improvement, in which I got rejected :(. 
+
+I started running unfaithfully and waiting for the Strimzi result, but it was too late, so I fell asleep hahaha.
```

Comment:
```suggestion
I started running unfaithfully and waiting for the Strimzi result, but it was too late, so I fell asleep.
```

I am not sure how the final "hahaha" would sound for a native english speaker. I would remove it.